lib/Core/SpecialFunctionHandler.h

```diff
@@ -14,6 +14,7 @@
 #include <map>
 #include <vector>
 #include <string>
+#include "TxTree.h"
```

Comment:
remove this if possible. Do we need this?

Reply:
this one is needed